src/main/java/seedu/address/MainApp.java

```diff
@@ -138,10 +142,30 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
         return initializedPrefs;
     }
 
+    /**
+     * Reads the filepath stored within a {@cide Storage} object and returns a {@code ReadOnlyAddressBook}.
+     *
+     * @param storage the storage object.
+     * @return a read-only address book.
+     */
+    private ReadOnlyAddressBook initAddressBook(Storage storage) {
+        ReadOnlyAddressBook initialAddressBook;
+        try {
+            initialAddressBook = storage.readInitialAddressBook();
+            initialWelcomeMessage = "Data file loaded successfully.";
+        } catch (DataLoadingException e) {
+            initialAddressBook = new AddressBook();
+            initialWelcomeMessage = "WARNING!! Entering a command will override the old data file.\n"
+                    + e.getMessage();
+        }
+        return initialAddressBook;
+    }
+
     @Override
     public void start(Stage primaryStage) {
-        logger.info("Starting AddressBook " + MainApp.VERSION);
```

Comment:
Should the version number be removed? It would be useful for debugging purposes in real-world scenarios.

Reply:
Re-added